ios_framework/Sources/LibHelper.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  LibHelper.swift
+//  Pods
+//
+//  Created by Dmitriy Shamaev on 21.03.2022.
+//
+
+import Foundation
+
+final class LibHelper {
+    static let version = "2.13-6"
```

Comment:
We need to support this version with PJ_VERSION_NUM_EXTRA somehow. May be we should add some here and near with PJ_VERSION_NUM_EXTRA ?

Reply:
Looks like this file can be removed. Originally it was required to fix dynamic linking. But since now we link PJSIP as static library, we don't need this file anymore.